Makefile

```diff
@@ -112,7 +112,7 @@ scan-platform: check-env
 	done;
 	for component in ${VENDOR_COMPONENTS} ; do \
 		PUSH_IMAGE=${REPOSITORY}/ap-$${component} \
-		PUSH_TAGS="${ASTRONOMER_VERSION}" \
+		PUSH_TAGS="${ASTRONOMER_REF}" \
```

Comment:
@andriisoldatenko should we update the `${ASTRONOMER_VERSION}` in the platform components to be `${ASTRONOMER_REF}` too?

Reply:
@schnie done.